src/Socket.cpp

```diff
@@ -720,6 +753,14 @@ int Socket::Send(const void* buffer, int length)
     return send(socket_, reinterpret_cast<const char*>(buffer), length, 0);
 }
 
+int Socket::Send(const void* const buffer, const int length, const PeerAddr& addr)
+{
+    if (status_ == INVALID)
+        return -1;
+
+    return sendto(socket_, reinterpret_cast<const char*>(buffer), length, 0, addr.GetAddr(), addr.GetSize());

```

Comment:
Correct. However I left this to the caller that should notice the error code returned by calling this on a TCP connection. The limitation is also mentioned in the header